### ISSUE TITLE
No `dropbearconvert` on UDM Pro 2.4.7

### ISSUE THREAD

**Issue by tomudding** [unifi-utilities/unifios-utilities#504]
**Describe the bug**
When using the [`15-add-root-ssh-keys.sh`](https://github.com/unifi-utilities/unifios-utilities/blob/main/on-boot-script-2.x/examples/udm-files/on_boot.d/15-add-root-ssh-keys.sh) (2.x variant), the following fails:

https://github.com/unifi-utilities/unifios-utilities/blob/6314d7afea13a15987e8c543e3763c528cfbc645/on-boot-script-2.x/examples/udm-files/on_boot.d/15-add-root-ssh-keys.sh#L41-L43

With the following message:

> Converting SSH private key to dropbear format
> ./15-add-root-ssh-keys.sh: line 43: dropbearconvert: command not found

**To Reproduce**
1. Add the script and some SSH keys
2. Execute the script
3. Observe error of missing `dropbearconvert`

**Expected behavior**
The script not to produce an error because some command is not present.

**Screenshots**
![image](https://user-images.githubusercontent.com/4983571/221270718-887f51b2-aea4-4ce1-ae5a-8125264744c3.png)

**UDM Information**
 - Variant: UDMP
 - Firmware Version: 2.4.7
 - Controller Version: 7.2.97

**Additional context**
N/A

**Comment by tomudding** [comment]
This was also in GH-497 (second error).

**Comment by harrisonj94** [comment]
I am also experiencing this

**Comment by johntdyer** [comment]
> This was also in [GH-497](https://github.com/unifi-utilities/unifios-utilities/issues/497) (second error).

Shame on me for batching two errors in one issue :)

**Comment by Joeharrison94** [comment]
> > This was also in [GH-497](https://github.com/unifi-utilities/unifios-utilities/issues/497) (second error).
> 
> Shame on me for batching two errors in one issue :)

Do you know the reason for this error?

**Comment by johntdyer** [comment]
@Joeharrison94  nope, zero idea... thanks for opening another ticket to try and get some answers :)

**Comment by Joeharrison94** [comment]
@johntdyer any update on this issue. Thanks
